pages/courses/[id].js

```diff
@@ -49,22 +46,24 @@ export default function Course({ course, schools }) {
         <link rel="icon" href="/images/tps-logo-color.svg" />
       </Head>
       <Stack spacing={1}>
-        <Grid item xs={2}>
-          <Button variant="contained" onClick={() => router.back()}>
-            Go Back
-          </Button>
-        </Grid>
-        {renderSection('course name', name)}
-        {renderSection('description', description)}
+        <Typography variant="h1">{name}</Typography>
         {renderSection('prerequisites', pre_req_note)}
         {renderSection('credit_types', credit_types)}
         {renderSection('course_number', course_number)}
         {renderSection('credit_hours', courses_credit_hours)}
         {renderSection('department', department)}
-        {renderSection(
-          'schools',
-          schools.map((school) => school.name)
-        )}
+        {renderSection('description', description)}
+        <Typography variant="h6" sx={{ fontWeight: 'bold' }}>
+          Schools
+        </Typography>
+        {schools.map((school) => (
+          <Link
+            key={school.school_number}
+            href={`/schools/${school.school_number}`}
+          >
+            {school.name}
+          </Link>
```

Comment:
🙈 The links are a bit hard to see with the dark palette, and maybe just a link isn't the answer (maybe a chip/button that functions as a link).
<img width="216" alt="Screenshot 2022-12-19 at 10 10 51 AM" src="https://user-images.githubusercontent.com/33558504/208469808-8cd006b3-96b3-4f6a-9f21-4c3681a03076.png">

💡 Also, we could display the school names in several columns (for large screens) instead of just a single column to reduce page height. I think the current A-Z sorting is the right approach.